openAiClient.js

```diff
@@ -34,6 +41,10 @@ async function sendQuestionToOpenAI(questionFromUser) {
   }
 }
 
+sendQuestionToOpenAI("Create a crouse for Introduction to Psychology?").then(response => {
```

Comment:
should this test be in main/prod branch?

Reply:
Otherwise, it looks good! Thanks for fixing some variable names ;)

Reply:
Thanks, yes we will fix it!